distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date.enso

```diff
@@ -370,7 +371,7 @@ type Date
              from Standard.Base import Date, Time_Of_Day, Time_Zone
 
              example_to_time = Date.new 2020 2 3 . to_date_time Time_Of_Day.new Time_Zone.utc
-    to_date_time : Time_Of_Day -> Time_Zone -> Date_Time.Date_Time
+    to_date_time : Time_Of_Day -> Time_Zone -> Date_Time_Type
```

Comment:
Paths like `Foo.Bar` are not supported in type signatures right now

Reply:
Are all kinds of paths forbidden? Or is it just that _constructors_ cannot be treated as types?

I don't understand why the former syntactic restriction on paths would be placed? If I have a type in module `Standard.Base.Foo.Bar.Baz` and I import `Foo`, I don't see why I cannot refer to it by `Foo.Bar.Baz`? (As long as it is still a _type_ and not a _constructor_, of course)

Reply:
Thought Dmitry, was just saying we need to alias types for now as doesn't support path style currently. I'm assuming this is just a checker thing and hopefully a small issue for us for time being.

Reply:
I see, I think I'm wondering if it is a temporary limitation or just that we don't plan to support it at all.

If former then sounds all totally OK, if latter I'd like to reconsider.

Reply:
Yes should only be temporary.

Reply:
>  If I have a type in module Standard.Base.Foo.Bar.Baz and I import Foo, I don't see why I cannot refer to it by Foo.Bar.Baz?

@radeusgd it's a bug. I didn't have time to look into it. I'll create an issue and put a `TODO` note next to every such workaround.